isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -382,22 +468,13 @@ one time) may be useful depending on the complexity of your processing requireme
         Guide</a> for instructions on downloading and installing ISIS 2.
     </p>
 
-<h3>How do I install ISIS 3.5.2 or earlier?</h3>
+<h3>How do I install ISIS3.5.2 or earlier?</h3>
     <p>
          If you are looking for a version of ISIS3 prior to 3.5.3, please
```

Comment:
This should say "prior to 3.6.0" instead of prior to "3.5.3"

Reply:
Fixed it.